.github/workflows/rhel-9-4.yml

```diff
@@ -16,15 +16,15 @@ jobs:
             endsWith(github.event.comment.body, '/test-rhel-9-4-ami') ||
             endsWith(github.event.comment.body, '/test-rhel-9-4-arm-ami') ||
             endsWith(github.event.comment.body, '/test-rhel-9-4-simplified') ||
-            endsWith(github.event.comment.body, '/test-rhel-9-4-fdo-aio') ||
-            endsWith(github.event.comment.body, '/test-rhel-9-4-9to9') ||
-            endsWith(github.event.comment.body, '/test-rhel-9-4-8to9') ||
-            endsWith(github.event.comment.body, '/test-rhel-9-4-ignition') ||
             endsWith(github.event.comment.body, '/test-rhel-9-4-minimal') ||
+            endsWith(github.event.comment.body, '/test-rhel-9-4-arm-minimal') ||
```

Comment:
We can remove all of them and just keep `/test-rhel-9-4`.

Reply:
fixed